rdr_service/dao/participant_summary_dao.py

```diff
@@ -725,6 +735,20 @@ def update_enrollment_status(self, summary: ParticipantSummary, session, allow_d
                     revised_consent_time_list.append(response.authored_datetime)
         revised_consent_time = min_or_none(revised_consent_time_list)
 
+        wgs_sequencing_time = None
+        genomic_set_member = GenomicSetMemberDao.get_member_by_participant_id_with_session(
+            session=session,
+            participant_id=summary.participantId,
+            genome_type=config.GENOME_TYPE_WGS
+        )
+        if genomic_set_member and genomic_set_member.qcStatus == GenomicQcStatus.PASS:
+            # Determine when the job ran to get the rough date of the pass
+            aw4_job_run: GenomicJobRun = GenomicJobRunDao.get_with_session(
+                session=session,
+                obj_id=genomic_set_member.aw4ManifestJobRunID
+            )
+            wgs_sequencing_time = aw4_job_run.startTime
+
```

Comment:
With this I think it might easier to build a util method in the genomic dao with the joins already instead of having to do multiple db calls here.

Reply:
Sounds good to me. Which dao should I use: GenomicSetMemberDao, GenomicJobRunDao, or another one?

Reply:
Since it deals with a member obj, I would use `GenomicSetMemberDao `.

Reply:
Sounds good. Updated in latest commit. Let me know what you think.